bciers/apps/reporting/src/app/components/activities/uiSchemas/gscNonCompressionNonProcessingUiSchema.ts

```diff
@@ -0,0 +1,228 @@
+import FieldTemplate from "@bciers/components/form/fields/FieldTemplate";
+import NestedArrayFieldTemplate from "@bciers/components/form/fields/NestedArrayFieldTemplate";
+import SourceTypeBoxTemplate from "@bciers/components/form/fields/SourceTypeBoxTemplate";
+import CheckboxWidgetLeft from "@bciers/components/form/widgets/CheckboxWidgetLeft";
+import InlineFieldTemplate from "@bciers/components/form/fields/InlineFieldTemplate";
+
+const uiSchema = {
+  "ui:FieldTemplate": FieldTemplate,
+  "ui:classNames": "form-heading-label",
+  gscWithProductionOfUsefulEnergy: {
```

Comment:
```suggestion
  gscWithProductionOfUsefulEnergy: {
```
gscFuelOrWasteLinearFacilitiesUsefulEnergy